NachoSpigot-Server/src/main/java/net/minecraft/server/EntityEnderPearl.java

```diff
@@ -78,6 +78,32 @@ protected void a(MovingObjectPosition movingobjectposition) {
                     location.setPitch(player.getLocation().getPitch());
                     location.setYaw(player.getLocation().getYaw());
 
+                    // Nacho start - Anti ender pearl glitch
+
+                    if (Nacho.get().getConfig().antiEnderPearlPatch) {
```

Comment:
Could you please fix the merge conflicts?

Reply:
If it was just the configuration I removed it for the moment